feature/gnoi/system/tests/copying_debug_files_test/copying_debug_files_test.go

```diff
@@ -53,28 +68,61 @@ func TestMain(m *testing.M) {
 //    https://github.com/fullstorydev/grpcurl
 //
 
+// gNOIKillProcess kills a daemon on the DUT, given its name and pid.
```

Comment:
I think the name and doc of the function should replace "kill" with "restart," because it always provides restart=true.

Reply:
Resolved in latest commit